meta-oe/recipes-printing/qpdf/qpdf_9.1.0.bb

```diff
@@ -0,0 +1,33 @@
+DESCRIPTION = "PDF transformation/inspection software"
+HOMEPAGE = "http://qpdf.sourceforge.net"
+LICENSE = "Artistic-2.0"
+SECTION = "libs"
+DEPENDS = "libpcre zlib libjpeg-turbo"
+
+SRC_URI = "https://github.com/qpdf/qpdf/archive/release-qpdf-${PV}.tar.gz"
+
+LIC_FILES_CHKSUM = "file://Artistic-2.0;md5=7806296b9fae874361e6fb10072b7ee3"
+SRC_URI[md5sum] = "9681c3ceb25040a7ce381196ddac937b"
+SRC_URI[sha256sum] = "98364eb3fec1bdc4343c7708100624e5fd2b3ada8e52532ee3480e8265bd5c5c"
+
+inherit autotools-brokensep gettext
+
+# disable random file detection for cross-compile
+EXTRA_OECONF = "--without-random \
+                --disable-static \
+                --disable-check-autofiles \
+                "
+
+EXTRA_OEMAKE_class-target = "LIBTOOL=${STAGING_BINDIR_CROSS}/${HOST_SYS}-libtool"
```

Comment:
perhaps you can use `LIBTOOL=${HOST_SYS}-libtool`

Reply:
Changed as requested.